tests/e2e/specs/test.js

```diff
@@ -37,12 +37,12 @@ describe("Browsing Categories", () => {
   it("Click Activity in Category", () => {
     cy.contains("Category: Errand");
     cy.contains(".activity-name", "Dog Walking").click();
-    cy.contains("Skip").click();
     cy.contains("Dog");
-    cy.contains("risk level");
+    cy.contains("risk");
   });
 });
 
+/*
```

Comment:
I commented out the following "Profile" test because it doesn't seem to exist on the phase2 version of the site. Please correct me if I am wrong

Reply:
If it doesn't exist, I can just delete this whole test as well

Reply:
I think it exists if you're coming to the page either already having filled out the profile or having already skipped, but I can confirm later. Either way, I don't think it matters since we're getting rid of the profile anyway. This updated test will pass once we get rid of the profile.

Reply:
yes, I was right. You can check by going to the staging website in an incognito window.

Reply:
Ah, okay, let me try that